package.json

```diff
@@ -48,9 +48,10 @@
     "serveDistLocalDb": "./end-to-end-test/local/runtime-config/serve_dist.sh",
     "testTime": "start_time=`date +%s` && karma start karma.conf.js && echo run time is $(expr `date +%s` - $start_time) s",
     "test": "yarn run testMain",
-    "testMain": "jest TableHeaderControls.spec.tsx --env=jsdom --runInBand --ci --reporters=default --reporters=jest-junit",
-    "testModulesCI": "JEST_JUNIT_UNIQUE_OUTPUT_NAME=true JEST_JUNIT_OUTPUT_DIR=/tmp/junit/module/ JEST_JUNIT_OUTPUT_NAME=unit-test-module yarn run testModules",
-    "testModules": "lerna run test --ignore=cbioportal-frontend --stream",
+    "check-env": "echo ",
+    "testMain": "jest $GREP --env=jsdom --runInBand --ci --reporters=default --reporters=jest-junit",
+    "testPackagesCI": "JEST_JUNIT_UNIQUE_OUTPUT_NAME=true JEST_JUNIT_OUTPUT_DIR=/tmp/junit/module/ JEST_JUNIT_OUTPUT_NAME=unit-test-module yarn run testPackages",
+    "testPackages": "lerna run test --ignore=cbioportal-frontend --stream",
```

Comment:
Hmm. We should probably rename `buildModules` and `publishModules` as well for consistency.  In that case we will also need to update https://github.com/cBioPortal/cbioportal-publisher/blob/master/.github/workflows/cbioportal-frontend-publish.yml#L27-L28.